openquake/qa_tests_data/scenario_risk/case_master/expected/losses_by_taxon-rlz-000.csv

```diff
@@ -0,0 +1,4 @@
+taxonomy,business_interruption,contents,nonstructural,occupants,structural,business_interruption_ins,contents_ins,nonstructural_ins,occupants_ins,structural_ins
+"tax1",8.48713E+04,5.26754E+05,7.59653E+05,NAN,3.93912E+05,8.48713E+04,5.26754E+05,7.59653E+05,NAN,3.93912E+05
```

Comment:
Yes, it is totally correct. There is no concept of insurance for occupants, so the column is left to NaN.